flink-runtime/src/main/java/org/apache/flink/runtime/metrics/ReporterSetup.java

```diff
@@ -120,55 +124,37 @@ public static ReporterSetup forReporter(String reporterName, MetricConfig metric
 	}
 
 	private static ReporterSetup createReporterSetup(String reporterName, MetricConfig metricConfig, MetricReporter reporter) {
-		LOG.info("Configuring {} with {}.", reporterName, metricConfig);
+		LOG.debug("Configuring {} with {}.", reporterName, metricConfig);
 		reporter.open(metricConfig);
 
 		return new ReporterSetup(reporterName, metricConfig, reporter);
 	}
 
-	public static List<ReporterSetup> fromConfiguration(final Configuration configuration) {
+	public static List<ReporterSetup> fromConfiguration(final Configuration configuration, final PluginManager pluginManager) {
+		LOG.debug("Initializing Reporters from Configuration: {}", configuration);
 		String includedReportersString = configuration.getString(MetricOptions.REPORTERS_LIST, "");
-		Set<String> includedReporters = reporterListPattern.splitAsStream(includedReportersString)
```

Comment:
reviews are a lot easier if we either a) refrain from non-critical refactorings b) move such refactorings into a separate commit.

Reply:
Done.

Reply:
Done.

Reply:
As this did not come up in the second round I consider the new split of commits as appropriate. Resolving.